en-GB/px-common.json

```diff
@@ -169,7 +169,7 @@
                     "min": "{{sessionLength}} minutes"
                 },
                 "scheduled": {
-                    "warning": "Note: if you can no longer make this time, please <a href='{{entranceLink}}'>cancel</a> or <a href='{{entranceLink}}'>reschedule</a> your participation as soon as possible.",
+                    "warning": "**Note:** if you can no longer make this time, please <a href='{{entranceLink}}'>cancel</a> or <a href='{{entranceLink}}'>reschedule</a> your participation as soon as possible.",
```

Comment:
Shouldn't this `a` be a markdown link?

Reply:
no, that's not a link, it's just supposed to be in bold

Reply:
the change that you did, yes, but apart from that there is a link in this string ans it should be move to markdown.

Reply:
but that was already there and was already working, is it really necessary to change it?

Reply:
done!